metaflow/plugins/kfp/tests/run_integration_tests.py

```diff
@@ -240,3 +283,7 @@ def exponential_backoff_from_platform_errors(
         raise MetaflowException(
             "KFAM issues not resolved after successive backoff attempts."
         )
+    kfp_run_id = re.search("Metaflow run_id=(.*)\n", run_and_wait_process.stderr).group(
```

Comment:
this is test code, but for production code, I highly recommend to stay away from regular expressions.